tests/test.collector.js

```diff
@@ -0,0 +1,67 @@
+import { default as Collector } from 'collector';
+
+
+describe('Collector', function() {
+
+  it('should be thrown an error if Message is created without a type', () => {
+    assert.throws(() => {
+      var Collection = new Collector();
```

Comment:
We should reserve variables starting with capital letters for `Classes` or `CONSTANTS`. I think it should be `var collection = new Collector();` here.

Makes it easier to keep track of what variable is of what type.